packages/launchpad/src/components/code/FileRow.vue

```diff
@@ -97,7 +98,7 @@ const props = defineProps<{
   description?: string
 }>()
 
-const openDocs = () => window.open('https://on.cypress.io/guides/configuration')
```

Comment:
I think we could just set the `href` prop on the Button and not have to worry about the right mutation here: https://github.com/cypress-io/cypress/blob/7c244f6b245a71685c019457bc76abdd626b123a/packages/frontend-shared/src/components/Button.vue#L122

Reply:
It will not be cool with router links...

Reply:
That's what the `internalLink`/`to` props are for 😛  lots of props on Button apparently

https://github.com/cypress-io/cypress/blob/7c244f6b245a71685c019457bc76abdd626b123a/packages/frontend-shared/src/components/Button.vue#L124

Reply:
... awwwww no! that's horrible.

I can do that... even if I am dying inside.

Reply:
I hate that `<Button>` takes a `href` but this seems like the way to go. We could fix this eventually all in one shot. Do you plan to do this @elevatebart ?

Reply:
I just did ;)